### PR TITLE
Client maktub develop ruan

### PR DESCRIPTION
**Pull request by Flame1994**

### CODE REVIEW

**Comment by angelomelonas** [comment]
Omg wou nou net se xD 22k additions kan nie reg wees nie xD

File: src/App.vue [hunk 1]
```diff
@@ -2,26 +2,68 @@
...
+				</v-row>
+			</div>
 		</v-main>
-		<!-- <footer-alpha />-->
 	</v-app>
 </template>
 
 <script lang="ts">
 import Vue from 'vue';
 import LandingSectionComponent from '@/core/components/LandingSectionComponent.vue';
```
Comment (angelomelonas) [inline src/App.vue]: Can we please get a rename on this ? :) `LandingSectionComponent` => `LandingSection`

File: src/App.vue [hunk 2]
```diff
@@ -2,26 +2,68 @@
...
 import AppBar from '@/core/components/AppBar.vue';
 import Link from '@/core/models/link';
 
 export default Vue.extend({
 	components: {
-		AppBar,
-		LandingSectionComponent
-		// FooterAlpha
+        AppBar,
+		LandingSectionComponent,
```
Comment (angelomelonas) [inline src/App.vue]: Just me, or is there some weird spacing here?

**Review by angelomelonas — CHANGES_REQUESTED** [review]
Looks good. Two small comments.

File: src/App.vue (hunk 2 above)
Comment (Flame1994) [reply src/App.vue]: Hmm looks fine on my IDE.

File: src/App.vue (hunk 1 above)
Comment (Flame1994) [reply src/App.vue]: Named it `SectionLanding` to keep with the naming convention.

**Review by angelomelonas — APPROVED** [review]

**Comment by angelomelonas** [comment]
@Flame1994 looks good - merge if you're happy.

**Pull request merged**